cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/processing/strategy/Strategy.kt

```diff
@@ -50,6 +50,11 @@ object Strategy {
         return x.nextEOG.iterator()
     }
 
+    /** A strategy to traverse the EOG in forward direction, but only if the edge is reachable. */
+    fun nextReachableEOG(x: Node): Iterator<Node> {
```

Comment:
should the name be harmonized with the rest? Like `REACHABLE_EOG_FORWARD`?

Reply:
In theory yes, although the uppercase naming is really bad. But we probably should rename all at the same time.